data/data-pipeline/data_pipeline/comparison_tool/src/tract_comparison__template.ipynb

```diff
@@ -0,0 +1,379 @@
+    "1. How many tracts are flagged and what's the size of overlap by comparator?\n",
+    "2. What are the demographics of each set of tracts by \"category\" of score (CEJST but not comparator, comparator but not CEJST, CEJST and comparator)?\n",
+    "3. What are the overall demographics of ALL comparator vs ALL CEJST?\n",
+    "\n",
+    "It produces a single Excel file of the stats listed, but is interactive even after run-time. This notebook focuses on 1:1 comparison. It can be pointed in the YAML to either a simple output (tract and boolean for highlight) or to the output from an ETL."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2093584e-312b-44f7-87d7-6099d9fe000f",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "## These are parameters and get overridden by the \"injected parameters\" cell below\n",
+    "ADDITIONAL_DEMO_COLUMNS = []\n",
+    "COMPARATOR_COLUMN = None\n",
+    "COMPARATOR_FILE = None\n",
+    "DEMOGRAPHIC_COLUMNS = []\n",
+    "DEMOGRAPHIC_FILE = None\n",
+    "OUTPUT_DATA_PATH = None\n",
+    "SCORE_FILE = None\n",
+    "OTHER_COMPARATOR_COLUMNS = None\n",
+    "OUTPUT_NAME = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bd38aaaa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## These are constants for all runs\n",
+    "GEOID_COLUMN = field_names.GEOID_TRACT_FIELD\n",
+    "SCORE_COLUMN = field_names.SCORE_M_COMMUNITIES\n",
+    "TOTAL_POPULATION_COLUMN = field_names.TOTAL_POP_FIELD"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a3940be",
+   "metadata": {},
+   "source": [
+    "__Date and time of last run__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f19a1e3-fe05-425d-9160-4d4e193fadf7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datetime.datetime.now()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "269b0f06-edee-4fce-8755-a8e7fe92e340",
+   "metadata": {},
+   "source": [
+    "__Congfigure output (autocreated)__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "acace944-18fa-4b05-a581-8147e8e09299",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "OUTPUT_EXCEL = os.path.join(\n",
+    "    OUTPUT_DATA_PATH,\n",
+    "    f\"{OUTPUT_NAME}__{datetime.datetime.now().strftime('%Y-%m-%d')}.xlsx\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5517217-f107-4fee-b4ca-be61f6b2b7c3",
+   "metadata": {},
+   "source": [
+    "__Read in data__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cdefecf2-100b-4ad2-b0bd-101c0bad5a92",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "comparator_cols = [COMPARATOR_COLUMN] + OTHER_COMPARATOR_COLUMNS if OTHER_COMPARATOR_COLUMNS else [COMPARATOR_COLUMN]\n",
+    "\n",
+    "#papermill_description=Loading_data\n",
```

Comment:
Just looked up what this does and that's sooooo nice!

Reply:
😄